### PR TITLE
fix: miss subscribers if llm service is turned off

### PR DESCRIPTION
**Pull request by tisfeng**
Fix https://github.com/tisfeng/Easydict/pull/599#issuecomment-2212466914

### CODE REVIEW

File: Easydict/Swift/Utility/GlobalContext.swift [hunk 1]
```diff
@@ -21,6 +21,12 @@ class GlobalContext: NSObject {
             updaterDelegate: updaterHelper,
             userDriverDelegate: userDriverHelper
         )
+
+        for service in services {
```
Comment (phlpsong) [inline Easydict/Swift/Utility/GlobalContext.swift:25]: Could this logic able to move inside `StreamConfigurationView` instead?

Comment (tisfeng) [reply Easydict/Swift/Utility/GlobalContext.swift:25]: No, we need to ensure that the LLM service is only subscribed once and remains unreleased, and it is appropriate to put it in the initialization method of this singleton.

Comment (phlpsong) [reply Easydict/Swift/Utility/GlobalContext.swift:25]: OK, this sounds reasonable.

I just concern the 'GlobalContext' sounds not the right place to do this. Maybe we could improve this in the future.

**Review by phlpsong — APPROVED** [review]
Issue fixed, good work👍

File: Easydict/Swift/Utility/GlobalContext.swift (hunk 1 above)
Comment (tisfeng) [reply Easydict/Swift/Utility/GlobalContext.swift:25]: Yes, this code should be improved later.

**Pull request merged**